var/spack/repos/builtin/packages/warpx/package.py

```diff
@@ -83,11 +85,14 @@ class Warpx(CMakePackage):
     variant('tprof', default=True,
             description='Enable tiny profiling features')
 
+    depends_on('sensei', when='+sensei')
+    depends_on('sensei +mpi', when='+sensei +mpi')
+
     depends_on('ascent', when='+ascent')
     # note: ~shared is only needed until the new concretizer is in and
     #       honors the conflict inside the Ascent package to find this
     #       automatically
-    depends_on('ascent +cuda ~shared', when='+ascent compute=cuda')
```

Comment:
This was fixed/not needed with ascent 0.8 anymore.

Reply:
Thanks, can you please also remove the comment above that documents why we needed `~shared`?